Booting/linux-bootstrap-1.md

```diff
@@ -1,29 +1,31 @@
-Kernel booting process. Part 1.
+Kernel Booting Process. Part 1.
 ================================================================================
 
-From the bootloader to kernel
+From the Bootloader to the Kernel
```

Comment:
Capitalising the nouns is normally how you format a title, if you would rather not I will change those back also.